go/pkg/command/command.go

```diff
@@ -585,6 +588,35 @@ func (c *Command) ToREProto(useOutputPathsField bool) *repb.Command {
 	return cmdPb
 }
 
+func FromREProto(cmdPb *repb.Command) *Command {
```

Comment:
This was moved here from tool.go, because it thematically belongs here, imo, next to the `ToREProto` method.